### ISSUE TITLE
[Bug][a11y] Selecting/deselecting an option to sync doesn't announce the option and the syncing action

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#16022]
## Steps to reproduce
1. Sign in with a valid account.
2. Turn voice assistant/talkback on.
3. Go to Settings - Account settings and select/deselect what to sync.

### Expected behavior
The voice assistant announces that selected/deselected option and the fact that the sync is started.

### Actual behavior
The voice assistant only announces "checked"/"not checked"; it doesn't read the option is being selected/deselected, and it doesn't announce that sync starts after selecting/deselecting an option.

### Device information
* Android device: Samsung Galaxy Tab S6 (Android 10)
* Fenix version: 10/19 Nightly, RC 82.1.0-RC.2

**Comment by mcarare** [comment]
This is the default, standard Talkback behavior for any two-state element. The content description is already announced when the element is selected. When the state is changed, only the new value (checked / unchecked, ticked / unticked) is announced. 
IMO this standard behavior should not be changed, as in the default behavior, any additional announcement other than the state change has a different relevance for the user (for example, changes in layout triggered by the state change, or actions triggered, like the "syncing" announcement in our case).
